src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationUnsuccessfulPostGateway.cshtml

```diff
@@ -0,0 +1,223 @@
+                                                            <div class="das-multiline-text-bottom-margin">@answer.Value</div>
+                                                            break;
+                                                    }
+                                                }
+                                            }
+
+                                        }
+                                        @if (Model.PagesWithGuidance != null && Model.PagesWithGuidance.Any(x => x.PageId == sectionPage.PageId))
+                                        {
+                                            var guidance = Model.PagesWithGuidance.FirstOrDefault(x => x.PageId == sectionPage.PageId);
+                                            @if (guidance?.GuidanceInformation != null && guidance.GuidanceInformation.Count > 0)
+                                            {
+                                                <div class="govuk-grid-row">
+                                                    <div class="govuk-grid-column-two-thirds">
+                                                        <details class="govuk-details govuk-!-margin-bottom-9">
+                                                            <summary class="govuk-details__summary">
+                                                                <span class="govuk-details__summary-text">
+                                                                    View application guidance
+                                                                </span>
+                                                            </summary>
+                                                            <div class="govuk-details__text">
+                                                                <ul class="govuk-list">
+                                                                    @foreach (var guidanceText in guidance.GuidanceInformation)
+                                                                    {
+                                                                        <li>
+                                                                            @Html.Raw(guidanceText)
+                                                                        </li>
+                                                                    }
+                                                                </ul>
+                                                            </div>
+                                                        </details>
+                                                    </div>
+                                                </div>
+                                            }
+                                        }
+                                    </span>
+                                }
+                            }
+
+                        }
+                    }
+                }
+            }
```

Comment:
The amount of braces is suggesting that we may want to break this view up into smaller components